src/simdvec.jl

```diff
@@ -435,6 +435,11 @@ for (op, llvmop) in [(:fma, Intrinsics.fma), (:muladd, Intrinsics.fmuladd)]
     end
 end
 
+if isdefined(Base, :bitrotate)
+    @inline Base.bitrotate(x::Vec, k::Vec) = Vec(Intrinsics.fshl(x.data, x.data, k.data))
+    @inline Base.bitrotate(x::Vec{N, T}, k::Integer) where {N, T} = Vec(Intrinsics.fshl(x.data, x.data, Vec{N, T}(k).data))
```

Comment:
I'd call the other `bitrotate` here; the code would be simpler.

Reply:
I updated the bitrotate that takes an Integer to call the bitrotate using the vector and squashed the change into the commit.